travis/travis_install_nightly

```diff
@@ -2,11 +2,6 @@
 
 pip install -q QUnitSuite flake8 coveralls pylint > /dev/null 2>&1
 
-# We can exit here and do nothing if this only a LINT check
-if [ "${LINT_CHECK}" == "1" ] ; then
-    exit 0
-fi
```

Comment:
The prevents from running lint ans server tests at the same time.
If uy«you are running only lint tests, this script isn't (or shouldn't be) called at all.